developer/doc_development/doc_development.tex

```diff
@@ -0,0 +1,461 @@
+
+\begin{itemize}
+    \item \alert{Examples}
+    \item \alert{API} (user manual)
+    \item \alert{Theory}
+\end{itemize}
+
+You may also want to check out:
+
+\begin{itemize}
+    \item \alert{Contribute} (developer guide)
+    \item \alert{Common use cases} (added in 2020)
+\end{itemize}
+\end{frame}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%
+% Theory            %
+%%%%%%%%%%%%%%%%%%%%%
+\section{Theory}
+
+% Outline
+\begin{frame}[label=tableofcontents]
+\frametitle{Outline}
+\tableofcontents[currentsection]
+\end{frame}
+
+
+\begin{frame}{Looking up the source file of a theory page}
+
+What holds for the \alert{Theory} section also mostly holds for the \alert{Contribute} (developer guide) and \alert{Common use cases} sections.
+
+Let us consider the \alert{Using QQ-plot to compare two samples} page:
+
+\url{https://openturns.github.io/openturns/master/theory/data_analysis/qqplot_graph.html}
+
+All Theory files are RST and the URL fits the source folder structure, so the relevant source file is:
+
+\href{https://github.com/openturns/openturns/blob/master/python/doc/theory/data_analysis/qqplot_graph.rst}{\texttt{python/doc/theory/data\_analysis/qqplot\_graph.rst}}
+
+There are great RST and Sphinx \href{https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html}{\alert{cheatsheet}} available,
+but here are the basics:
+
+\begin{itemize}
+    \item A page can be \alert{indexed} as \texttt{page\_name} by writing \texttt{.. \_page\_name:} at the top.
+    \item Because of this, that page can be \alert{linked} to from anywhere in the doc with \texttt{:ref:`page\_name`}
+    \item \alert{Titles} are underlined (the number of underlining characters must match the title).
+    \item \alert{Inline math} is \LaTeX{} code with \texttt{\$a=b\$} replaced with \texttt{:math:`a=b`}.
+    \item \alert{Display math} is \LaTeX{} code with \texttt{\$\$a=b\$\$} replaced with
+    
+    \texttt{.. math::}
+    
+    \texttt{~~~~a = b}
+    \item \alert{Macros} are defined in \href{https://github.com/openturns/openturns/blob/master/python/doc/math_notations.sty}{\texttt{python/doc/math\_notations.sty}}
+\end{itemize}
+
+\end{frame}
+
+
+\begin{frame}[fragile]{Adding a new theory page}
+Code of the main table of contents (\href{https://github.com/openturns/openturns/blob/master/python/doc/theory/theory.rst}{python/doc/theory/theory.rst}):
+
+\begin{lstlisting}
+.. _theory:
+
+======
+Theory
+======
+
+The theoretical documentation.
+This contains an in-depth description of all algorithms.
+
+.. toctree::
+    :maxdepth: 2
+
+    data_analysis/data_analysis.rst
+    probabilistic_modeling/probabilistic_modeling.rst
+    meta_modeling/meta_modeling.rst
+    reliability_sensitivity/reliability_sensitivity.rst
+    numerical_methods/numerical_methods.rst
+\end{lstlisting}
+Table of contents of subdirectories are essentially the same.
+\end{frame}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%
+% Examples          %
+%%%%%%%%%%%%%%%%%%%%%
+\section{Examples}
```

Comment:
1. A mention of the bibliography would be helpful: 

https://github.com/openturns/openturns/blob/master/python/doc/bibliography.rst

along with an example to use a reference:

https://github.com/openturns/openturns/blame/master/python/doc/theory/reliability_sensitivity/sensitivity_ancova.rst#L97

2. The writer should _not_ select his own set of notations, but rather use the notations already used in the Theory section. A new page describing a set of consistent notations should be created for this purpose. I suggest to use mine : 

https://github.com/mbaudin47/otsupgalilee-eleve/blob/SessionJanvier/0-Deroulement/Notations.md

but this might require some discussion.

Reply:
Added a mention concerning the biography. I agree your second point requires further discussion.